app/src/ui/check-runs/ci-check-run-popover.tsx

```diff
@@ -265,7 +269,20 @@ export class CICheckRunPopover extends React.PureComponent<
       }
     }
 
-    return <Donut valueMap={getCheckStatusCountMap(checkRuns)} />
+    const valueMap = getCheckStatusCountMap(checkRuns)
+
+    const ariaLabel = `Completeness indicator. ${
+      valueMap.get(APICheckStatus.Completed) ?? 0
+    } completed, ${valueMap.get(APICheckStatus.InProgress) ?? 0} in progress, ${
+      valueMap.get(APICheckStatus.Queued) ?? 0
+    } queued.`
+
+    return (
+      <Donut
+        ariaLabel={ariaLabel}
+        valueMap={valueMap}
+      />
+    )
```

Comment:
Sorry, my suggestion broke the linter 😳 
```suggestion
    return <Donut ariaLabel={ariaLabel} valueMap={valueMap} />
```

Reply:
:D no  worries, Going to pull it down to see about adding a tooltip to the window control buttons and I figured I would just catch it then. :)